### PR TITLE
feat(route): add route for https://jw.cdu.edu.cn/

### PR DESCRIPTION
**Pull request by uuwor**
## Involved Issue / 该 PR 相关 Issue

None #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/cdu/jwgg
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
This is my first attempt, and I hope you can point it out if there is any mistakes

### CODE REVIEW

**Comment by cxheng315** [comment]
@uuwor the file should be under `lib/routes` directory

**Comment by cxheng315** [comment]
@uuwor, your current file is in the `lib` directory instead of `lib/routes`. 
Please move the **entire folder** to the `/routes` sub-directory

**Review by TonyRL — COMMENTED** [review]
Please double-check the CSS selectors. 5/7 do not match anything

**Review by TonyRL — COMMENTED** [review]
https://github.com/DIYgod/RSSHub/pull/16548#discussion_r1732857231

File: lib/routes/cdu/namespace.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: '成都大学',
+    url: 'https://www.cdu.edu.cn/',
```
Comment (TonyRL) [inline lib/routes/cdu/namespace.ts:5]:
```suggestion
    url: 'www.cdu.edu.cn',
```

**Pull request merged**